BGL/include/CGAL/boost/graph/Graph_with_descriptor_with_graph.h

```diff
@@ -135,11 +139,18 @@ struct Graph_with_descriptor_with_graph
   typedef Gwdwg_descriptor<Graph, typename gt::edge_descriptor> edge_descriptor;
   typedef Gwdwg_descriptor<Graph, typename gt::face_descriptor> face_descriptor;
 
+  Graph_with_descriptor_with_graph()
+    : graph(NULL)
+  {}
+
   Graph_with_descriptor_with_graph(Graph& graph)
     : graph(&graph)
   {}
-private:
-  Graph_with_descriptor_with_graph(const Graph_with_descriptor_with_graph&){} // disable copy-constructor to avoid non-wanted copies
+  
+  // private:   
+  // disable copy-constructor to avoid non-wanted copies
+  // AF: Why was this private??
```

Comment:
It is supposed to be private, to keep the user from using it.

Problem is : Polyhedral_mesh_domain_with_features_3 requires a copy of the template type Polyhedron. If we disable this, what do we do for surface_only remeshing ?